src/full_node/full_node.py

```diff
@@ -409,7 +387,8 @@ async def _sync(self):
                 return
 
             fork_point_height = self.sync_store.get_potential_fork_point(heaviest_peak_hash)
-
+            # cache warmup to fork height
+            await self.blockchain.warmup(fork_point_height)
```

Comment:
We need to warmup again after finishing sync. It's possible we fail syncing without extending the peak

Reply:
not sure i get why,
let's say we start syncing from forkpoint x and fail at x+100, the cache state would now be as if we called self.blockchain.warmup(fork_point_height +100). 
calling warmup to forkpoint doesn't remove the sub_blocks that are above the forkpoint as we said the current sync logic we can still be working on the current peak and trying to sync to a new one at the same time.